leptos_macro/src/view.rs

```diff
@@ -1831,23 +1831,13 @@ fn is_custom_element(tag: &str) -> bool {
 fn is_self_closing(node: &NodeElement) -> bool {
     // self-closing tags
     // https://developer.mozilla.org/en-US/docs/Glossary/Empty_element
-    matches!(
-        node.name().to_string().as_str(),
-        "area"
-            | "base"
-            | "br"
-            | "col"
-            | "embed"
-            | "hr"
-            | "img"
-            | "input"
-            | "link"
-            | "meta"
-            | "param"
-            | "source"
-            | "track"
-            | "wbr"
-    )
+    // Keep list alphabetized for binary search
```

Comment:
Is this actually an improvement? I would've thought that matching on a set list of names would be faster than the search here. I may be wrong, of course. 

ETA: Same applies to the SVG and MathML searches. Although now that I think about it a little more, I guess it's an `O(n)` conditional here (`if name == X else if name == Y else if name == Z`) so a binary search is better, you're right.